src/app.js

```diff
@@ -1 +1,104 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from 'axios';
+console.log('Hallo daar!');
+
+// Schrijf een asynchrone functie die een GET request maakt naar endpoint /all
+
+async function fetchAllCountries () {
+    try {
+        const result = await axios.get("https://restcountries.com/v2/all");
+        console.log(result)
+        // Hier maak ik een array van alleen de data van de landen
+        const allCountriesArray = result.data;
+        // Hier sorteer ik die array op basis van populatie van laag naar hoog
+        const allCountriesSortByPop = allCountriesArray.sort((a, b) =>  {
+            return a.population - b.population;
+        });
+        // Map functie, waarin ik een array maak met alle html-elementen die geprint moeten worden per land. Elk <li> element heeft een class gebaseerd op de regio van het land, daarom is de aparte functie voor kleuren afwezig.
+        const displayAllCountries = allCountriesSortByPop.map((country, index) => {
```

Comment:
Hier zou je spreading kunnen toepassen, zodat je niet op meerder plekken `country.` vóór hoeft te zetten
```suggestion
        const displayAllCountries = allCountriesSortByPop.map(({ region, flag, name, population }, index) => {
```